src/gateway/kong-production/monitoring/prometheus.md

```diff
@@ -1,6 +1,130 @@
 ---
-title: Prometheus How to monitor
-
+title: Monitoring with Prometheus
+content_type: how-to
 ---
 
-## PLACEHOLDER Monitoring with Prometheus
\ No newline at end of file
+[Prometheus](https://prometheus.io/) is a popular systems monitoring and alerting toolkit. Prometheus 
+implements a multi-dimensional time series data model and distributed storage
+system where metrics data is collected via a pull model over HTTP. 
+
+{{site.base_gateway}} supports Prometheus with the [Prometheus Plugin](/hub/kong-inc/prometheus/) that exposes
+{{site.base_gateway}} performance and proxied upstream service metrics on the `/metrics` endpoint.
+
+This guide will help you setup a test {{site.base_gateway}} and
+Prometheus service. Then you will generate sample requests to {{site.base_gateway}} and
+observe the collected monitoring data.
+
+### Prerequisites
+This guide assumes the following tools are installed locally:
+* [Docker](https://docs.docker.com/get-docker/) is used to run {{site.base_gateway}}, the supporting database, 
+and Prometheus locally. 
+* [curl](https://curl.se/) is used to send requests to {{site.base_gateway}}. `curl` is pre-installed on most systems.
+
+### Configure Prometheus monitoring
+
+1. Run the following command to start {{site.base_gateway}} using Docker:
```

Comment:
Lets make this resemble the statsd doc step1:
https://github.com/Kong/docs.konghq.com/pull/4260/commits/86f6ab9ce198296ae1c8b663f5587874a98ad0f8#diff-fa78962aed3b14a6e451b72d353810e20572ecba283cdb6e404e0558c1af1160R20